app/assets/javascripts/backbone/views/error_handler.coffee

```diff
@@ -0,0 +1,25 @@
+class MS.Views.ErrorHandler extends Backbone.View
+
+  el: 'body'
+
+  initialize: ->
+    MS.loadingIndicator = @$('#glb-load-ind')
```

Comment:
@hooverlunch I don't think it can move... Otherwise, we have no way to hide the loading indicator when the error happens.